<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddStudentWidget</class>
 <widget class="QDialog" name="AddStudentWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>311</width>
     <height>347</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="namelabel">
      <property name="text">
       <string>姓名:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="idlabel">
      <property name="text">
       <string>学号:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="idlineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="agelabel">
      <property name="text">
       <string>年龄:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="agelineEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="genlabel">
      <property name="text">
       <string>性别:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="comboBox">
      <item>
       <property name="text">
        <string>男</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>女</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="tellabel">
      <property name="text">
       <string>电话:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="tellineEdit"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="birlabel">
      <property name="text">
       <string>生日:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="birlineEdit">
      <property name="inputMask">
       <string/>
      </property>
      <property name="placeholderText">
       <string>YYYYMMDD</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="addresslabel">
      <property name="text">
       <string>地址:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="addresslineEdit"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="gradelabel">
      <property name="text">
       <string>成绩:</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="gradelineEdit"/>
    </item>
    <item row="8" column="1">
     <widget class="QPushButton" name="addButton">
      <property name="text">
       <string>添加</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QPushButton" name="returnButton">
      <property name="text">
       <string>主菜单</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="namelineEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
